Resources/Prototypes/Nyanotrasen/Entities/Objects/Weapons/Guns/Snipers/snipers.yml

```diff
@@ -0,0 +1,49 @@
+- type: entity
+  name: Mark 1 Rifle
+  parent: BaseWeaponSniper
+  id: WeaponSniperGrand
+  description: An ancient semi-automatic rifle that sparks some kind of a deep feeling from within you. Uses .30 rifle ammo.
```

Comment:
Telling the reader what they feel is generally bad writing. Write to evoke the feeling.

Reply:
I'll write up something a little nicer.